parsec/core/gui/main_window.py

```diff
@@ -483,6 +483,17 @@ def switch_to_tab(self, idx):
         if not ParsecApp.has_active_modal():
             self.tab_center.setCurrentIndex(idx)
 
+    def switch_to_login_tab(self):
+        idx = self._get_login_tab_index()
+        if idx != -1:
+            self.switch_to_tab(idx)
+        else:
+            tab = self.add_new_tab()
+            tab.show_login_widget()
+            self.on_tab_state_changed(tab, "login")
+            self.switch_to_tab(self.tab_center.count() - 1)
+            idx = self.tab_center.count() - 1
```

Comment:
What's the need for getting `idx` here ?

Reply:
it's pointless, i removed it